src/Connections/GenericHook.ts

```diff
@@ -432,6 +432,7 @@ export class GenericHookConnection extends BaseConnection implements IConnection
                 hookId: this.hookId
             }
         );
+        this.state = validatedConfig;
```

Comment:
what, how, what, how did this work

Reply:
Should it not...? `state` and `validatedConfig` are each a `GenericHookConnectionState`.